tests/v2/test_canditate_register.py

```diff
@@ -0,0 +1,148 @@
+"""Tests candidates endpoint """
+import json
+import unittest
+
+from app import app
+from config import app_config
+from app.api.v2.models.db import init_db
+
+import jwt
+import os
+KEY = os.getenv('SECRET_KEY')
+
+
+class BaseTestClass(unittest.TestCase):
+    """
+    Setting up tests
+    """
+
+    def setUp(self):
+        self.app = app("testing")
+        self.client = self.app.test_client()
+        self.app.config['TESTING'] = True
+        init_db()
+
+        self.admintoken = "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJlbWFpbCI6InRldmludGh1a3VAZ21haWwuY29tIn0.Nehevl-RbT5FFF484k1fuW9DV7u5ZqrgiPqAe7igpwA"
```

Comment:
This is a potential security issue, You can add it to the `.env`

Reply:
I have placed the secure information in the .env and added a secure env variable to Travis CI so that it gets the variable when running tests that require the admin to do certain operations.